flocker/node/test/test_deploy.py

```diff
@@ -1573,9 +1573,13 @@ def test_volume_handoff(self):
         # application is running here.
         current = Deployment(nodes=frozenset([node, another_node]))
 
+        volume_service = create_volume_service(self)
+        self.successResultOf(volume_service.create(
+            volume_service.get(_to_volume_name(DATASET_ID))))
+
```

Comment:
I'm not suggesting changing it here, but I found this, and the other updated tests, difficult to read. It'd be nice to use variable names like `node1`, `node2`, `node1_volume_service`, `node1_deployer` to make it clear what situation we're trying to simulate.